project_national/fact_sheets_category_6.yml

```diff
@@ -0,0 +1,979 @@
+      - *fuel_oil_boiler_secondary_heating_type_fuel
+      - *secondary_boiler_76
+      - *secondary_boiler_80
+      - *secondary_boiler_90
+      - *fuel_oil_wallfloor_furnace_secondary_heating_type_fuel
+      - *secondary_wallfloor_furnace_60
+      - *secondary_wallfloor_furnace_68
+      - *cooling_setpoint_no_offset
+      - *cooling_setpoint_offset_magnitude_0     
+      - *cooling_setpoint_offset_period_none
+      - *heating_setpoint_no_offset
+      - *heating_setpoint_offset_magnitude_0
+      - *heating_setpoint_offset_period_none
+
+
+ 
+
+      #### ducted furnace ####
+      - &ducted_ASHP_HigherEfficiency_exist_backup_shared_ducts_60_FO
+        option: HVAC Heating Efficiency|Dual-Fuel ASHP, SEER 20, 11 HSPF, CCHP, Integrated Backup, 60% AFUE Fuel Oil, 5F-40F switchover band, Max Load 
+        apply_logic: *dual-fuel_60pctAFUE_FO_furnace_apply_logic   
+        costs: *costs_ashp_ducted_SEER_20_11_HSPF
+        lifetime: *lifetime_ashp_ducted_SEER_20_11_HSPF
+      - &ducted_ASHP_HigherEfficiency_exist_backup_shared_ducts_76_FO
+        option: HVAC Heating Efficiency|Dual-Fuel ASHP, SEER 20, 11 HSPF, CCHP, Integrated Backup, 76% AFUE Fuel Oil, 5F-40F switchover band, Max Load 
+        apply_logic: *dual-fuel_76pctAFUE_FO_furnace_apply_logic     
+        costs: *costs_ashp_ducted_SEER_20_11_HSPF
+        lifetime: *lifetime_ashp_ducted_SEER_20_11_HSPF
+      - &ducted_ASHP_HigherEfficiency_exist_backup_shared_ducts_80_FO
+        option: HVAC Heating Efficiency|Dual-Fuel ASHP, SEER 20, 11 HSPF, CCHP, Integrated Backup, 80% AFUE Fuel Oil, 5F-40F switchover band, Max Load 
+        apply_logic: *dual-fuel_80pctAFUE_FO_furnace_apply_logic     
+        costs: *costs_ashp_ducted_SEER_20_11_HSPF
+        lifetime: *lifetime_ashp_ducted_SEER_20_11_HSPF
+      - &ducted_ASHP_HigherEfficiency_exist_backup_shared_ducts_92-5_FO
+        option: HVAC Heating Efficiency|Dual-Fuel ASHP, SEER 20, 11 HSPF, CCHP, Integrated Backup, 92.5% AFUE Fuel Oil, 5F-40F switchover band, Max Load 
+        apply_logic: *dual-fuel_92-5pctAFUE_FO_furnace_apply_logic
+        costs: *costs_ashp_ducted_SEER_20_11_HSPF
+        lifetime: *lifetime_ashp_ducted_SEER_20_11_HSPF
+
+  - upgrade_name: EnergyStar ductless MSHP w/ existing natural gas backup  
+    # Measure 4
+    package_apply_logic: &dualfuel_MSHP_apply_logic_NG
+      - HVAC Has Ducts|No
+      - Heating Fuel|Natural Gas
+      - not:
+          or:
+            - HVAC Heating Efficiency|Shared Heating
+            - HVAC Cooling Efficiency|Shared Cooling  
+    options:
+      # MSHP
+      - &ductless_MSHP_EnergyStar_exist_backup
+        option: HVAC Heating Efficiency|MSHP, SEER 16, 9.2 HSPF, Separate Backup, Max Load  
+        apply_logic: &ductless_MSHP_exist_backup_apply_logic
+          - or:
+            - not: HVAC Cooling Efficiency|Non-Ducted Heat Pump
+            - not: HVAC Cooling Efficiency|Ducted Heat Pump
+            - HVAC Heating Efficiency|MSHP, SEER 14.5, 8.2 HSPF
```

Comment:
NIT, this apply_logic block will never apply for this yaml because it's for non-electric fuels. Can remove